sklearn/datasets/_svmlight_format_io.py

```diff
@@ -85,7 +85,7 @@ def load_svmlight_file(
 
     Parameters
     ----------
-    f : str, file-like or int
+    f : str, path-like, file-like or int
```

Comment:
May we add a `.. versionadded:: 1.2` directive under the docstring for `f` that states "Adds path-like object support"?

Reply:
I added `versionchanged` instead, which seemed more fitting, but the proposed changes are done.

Reply:
I agree `versionchanged` is better. Thanks for the update!